src/main/java/mate/jdbc/dao/CarDaoImpl.java

```diff
@@ -0,0 +1,223 @@
+                + "SET model = ?, manufacturer_id = ? "
+                + "WHERE id = ? AND is_deleted = FALSE";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement
+                        = connection.prepareStatement(query)) {
+            statement.setString(1, car.getModel());
+            statement.setLong(2, car.getManufacturer().getId());
+            statement.setLong(3, car.getId());
+            statement.executeUpdate();
+        } catch (SQLException e) {
+            throw new DataProcessingException("Couldn't update "
+                    + car + " in carsDB", e);
+        }
+        deleteDrivers(car);
+        insertDrivers(car);
+        return car;
+    }
+
+    @Override
+    public boolean delete(Long id) {
+        String query = "UPDATE cars SET is_deleted = TRUE WHERE id = ?";
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement statement
+                         = connection.prepareStatement(query)) {
+            statement.setLong(1, id);
+            return statement.executeUpdate() > 0;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Couldn't delete car with id " + id, e);
+        }
+    }
+
+    @Override
+    public List<Car> getAllByDriver(Long driverId) {
+        String query = "SELECT cars.id AS cars_id, cars.model AS cars_model, "
+                + "manufacturer.id AS manufacturer_id, manufacturer.name AS manufacturer_name,"
+                + "manufacturer.country AS manufacturer_country "
+                + "FROM cars_drivers "
+                + "JOIN cars ON cars_drivers.car_id = cars.id "
+                + "JOIN manufacturers ON cars.manufacturer_id = manufacturers.id "
+                + "WHERE cars_drivers.driver_id = ? AND cars.is_deleted = false "
+                + "AND manufacturers.is_deleted = false";
+        List<Car> cars = new ArrayList<>();
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(query)) {
+            statement.setLong(1, driverId);
+            ResultSet resultSet = statement.executeQuery();
+            while (resultSet.next()) {
+                cars.add(getCar(resultSet));
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Couldn't get cars for " + driverId, e);
+        }
+        cars.forEach(car -> car.setDrivers(selectDrivers(car)));
+        return cars;
+    }
+
+    private Car getCar(ResultSet resultSet) throws SQLException {
+        final Long carId = resultSet.getObject("cars.id", Long.class);
+        final String model = resultSet.getString("cars.model");
+        Manufacturer manufacturer = new Manufacturer();
+        manufacturer.setId(resultSet.getObject("manufacturers.id", Long.class));
+        manufacturer.setName(resultSet.getString("manufacturers.name"));
```

Comment:
Incorrect column names. Change to: 'manufacturers_id', 'manufacturers_name', and 'manufacturers_country'